### ISSUE TITLE
Enable debugger in docker image

### ISSUE THREAD

**Issue by iiLaurens** [browserless/browserless#4072]
**Is your feature request related to a problem? Please describe.**
For local installations and local development the debugger has been crucial, and has been the main reason why I have not upgrade to v2 yet. Fortunately #4024 merged debugger support. However currently it appears the debugger is not enabled in the docker images. Could you consider installing this by default or distributing an additional image tag that has this enabled?

Currently I can enable the debugger by running `npm run install:debugger` inside the container. But I am assuming this needs to be done every time the base image gets updated.

**Comment by 100terres** [comment]
For now, on our side we've had to create a `Dockerfile` and use it in our `compose.yaml` file.
```dockerfile
FROM ghcr.io/browserless/chromium:v2.12.0
RUN npm run install:debugger
```

**Comment by joelgriffith** [comment]
Yup, we'll fix this, should be easy enough

**Comment by joelgriffith** [comment]
Should be fixed in https://github.com/browserless/browserless/pull/4076

**Comment by white0dew** [comment]
> Should be fixed in [#4076](https://github.com/browserless/browserless/pull/4076)应固定在 [#4076](https://github.com/browserless/browserless/pull/4076)

请问解决了吗？我现在还是无法使用dbugger
